juntagrico/entity/subs.py

```diff
@@ -149,8 +150,11 @@ def required_shares(self):
         return result
 
     def recipients_names(self):
-        members = self.recipients
-        return ', '.join(str(member) for member in members)
+        if self.nickname:
```

Comment:
da diese methode an anderen stellen ausser der depotliste benutzt wird sollte die nicht angepasst werden vorallem macht sie nun nicht mehr das was sie sagt
am besten eine neue methode machen wie nickname_or_other_recipients
welche den nickname oder die other recipients aufruft. wie schon erwähnt der primary member ist vom nickname nicht betroffen. diese neue methode dann in der depotliste verwenden